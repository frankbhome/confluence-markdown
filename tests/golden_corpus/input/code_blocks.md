# Code Blocks Example

## Fenced Code Block with Language

```python
def hello_world():
    print("Hello, World!")
    return True
```

## Fenced Code Block without Language

```
This is a plain code block
without any language specified.
```

## Inline Code

Here is some `inline code` in a sentence.

## Mixed Content

The function `calculate()` works like this:

```javascript
function calculate(a, b) {
    return a + b;
}
```
