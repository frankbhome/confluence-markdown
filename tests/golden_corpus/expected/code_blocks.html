<h1>Code Blocks Example</h1>

<h2>Fenced Code Block with Language</h2>

<ac:structured-macro ac:name="code" ac:schema-version="1"><ac:parameter ac:name="language">python</ac:parameter><ac:plain-text-body><![CDATA[def hello_world():
    print("Hello, World!")
    return True]]></ac:plain-text-body></ac:structured-macro>

<h2>Fenced Code Block without Language</h2>

<ac:structured-macro ac:name="code" ac:schema-version="1"><ac:parameter ac:name="language">text</ac:parameter><ac:plain-text-body><![CDATA[This is a plain code block
without any language specified.]]></ac:plain-text-body></ac:structured-macro>

<h2>Inline Code</h2>

<p>Here is some <code>inline code</code> in a sentence.</p>

<h2>Mixed Content</h2>

<p>The function <code>calculate()</code> works like this:</p>

<ac:structured-macro ac:name="code" ac:schema-version="1"><ac:parameter ac:name="language">javascript</ac:parameter><ac:plain-text-body><![CDATA[function calculate(a, b) {
    return a + b;
}]]></ac:plain-text-body></ac:structured-macro>
